Failures occuring in tests. Kind of a weird one.

```
[----------] 4 tests from Configurations/AudioTest
unknown file: Failure
C++ exception with description "basic_string: construction from null is not valid" thrown in SetUpTestSuite().

unknown file: Failure
C++ exception with description "basic_string: construction from null is not valid" thrown in auxiliary test code (environments or event listeners).

/__w/Sunshine/Sunshine/pkg/PKGBUILD: line 181:  9195 Aborted                    (core dumped) ./test_sunshine --gtest_color=yes --gtest_output=xml:test_results.xml
==> ERROR: A failure occurred in check().
    Aborting...
Error: Process completed with exit code 4.
```

I'm running your other PR to see if it's caused by this or just a new issue in general.

--- a/src/platform/linux/misc.cpp
+++ b/src/platform/linux/misc.cpp
@@ -445,8 +445,22 @@ namespace platf {
     lifetime::exit_sunshine(0, true);
   }
 
+  std::string get_env(const std::string &name) {
+    if (const auto value = getenv(name.c_str()); value != nullptr) {
+      return value;
+    }
+    return "";
+  }
+
   int set_env(const std::string &name, const std::string &value) {
     return setenv(name.c_str(), value.c_str(), 1);
+  }
+
+  int append_env(const std::string &name, const std::string &value, const std::string &separator) {
+    if (const std::string old_value = get_env(name); !old_value.contains(value)) {
+      return set_env(name, old_value.empty() ? value : old_value + separator + value);
+    }
+    return 0;
   }
 
   int unset_env(const std::string &name) {
@@ -1130,6 +1144,8 @@ namespace platf {
 
     // enable Vulkan video extensions for AMD RADV
     set_env("RADV_PERFTEST", "video_encode");
+    // Above is deprecated on Mesa 26.1+ and replaced by (keep both to ensure best compatibility):
+    append_env("RADV_EXPERIMENTAL", "video_encode", ",");
 
     // These are allowed to fail.
     gbm::init();

